frontend/packages/console-app/src/utils/isMultiClusterEnabled.ts

```diff
@@ -0,0 +1,4 @@
+const isMultiClusterEnabled = (): boolean => window.SERVER_FLAGS.clusters?.length > 1;
```

Comment:
Is this valid?
What if the hub cluster only has a one managed cluster? Won't this array contain a single item?

Should we be checking for undefined or null instead?

Reply:
The cluster list will always include the hub cluster

Reply:
Thanks for the clarification.

Ok so if there's one cluster it would mean that we are always targeting the hub cluster in which case the check is correct.

While maybe not correct for the web terminal which can work to access the cluster the console runs on. But that's another discussion.